lib/localenv/localenv.go

```diff
@@ -543,7 +543,6 @@ func (env *LocalEnvironment) AppServiceLocal(config AppConfig) (service appbase.
 		DockerClient: dockerClient,
 		ImageService: imageService,
 		StateDir:     filepath.Join(env.StateDir, "import"),
-		Devmode:      env.Debug,
```

Comment:
This was placing the `gravity-site` into development mode in `gravity --debug site import`.